vuepress/docs/docs/federation.md

```diff
@@ -0,0 +1,81 @@
+# Federation
```

Comment:
You also need to add this page in https://github.com/ghostdogpr/caliban/blob/master/vuepress/docs/.vuepress/config.js (at the bottom)

Reply:
Ahh yeah I was wondering how it got indexed